### DISCUSSION TITLE
Color blindness plugin

### DISCUSSION THREAD

**Discussion by VALTZC** [runelite/runelite#17682] [Ideas]
A color blindness plugin with settings you can mess with to match to your preference. The one there is changes it too much for me.

**Comment by Nightfirecat** [comment]
The GPU plugin has a number of colorblindness mode filters. Is that what you're referring to?

**Reply by VALTZC** [reply]
Yes, there are couple of filters. but they are too strong for my eyes. A some sort of slider to change the intensity would be nice, like in 117  plugin you can change it. (Dont want to use it bc HD bad)

**Reply by bedmndsn** [reply]
Just bumping this idea. Came here to suggest the exact same thing for the exact same reasons. Would make the GPU Colorblindness correction filter actually useable.

**Reply by bedmndsn** [reply]
I did some poking around and have a working version of this!
#19882

**Reply by ganztax** [reply]
Huge for the accessibility crowd.
